pkg/client/speak/constants.go

```diff
@@ -6,10 +6,39 @@ package speak
 
 import (
 	"errors"
+	"time"
+)
+
+// internal constants for retry, waits, back-off, etc.
+const (
```

Comment:
Delete this `const` block. None of these are being used.

Reply:
The defaultDelayBetweenRetry is being used here: https://github.com/deepgram/deepgram-go-sdk/blob/8f388b7bd0cd40c95c1ba4e99da39f5b368e65d0/pkg/client/speak/client_stream.go#L217

I've left that one for now, unless you think that logic in internalConnectWithCancel is unneeded

Reply:
Ooops! I missed that. It looks like all of them based on the golintci feedback